src/Html/Builder.php

```diff
@@ -116,9 +116,9 @@ public function generateScripts()
         $args = array_merge(
             $this->attributes, [
                 'ajax'    => $this->ajax,
-                'columns' => $this->collection->map(function ($column) {
-                    unset($column->attributes);
-
+                'columns' => $this->collection->map(function (Column $column) {
+                    $column=$column->toArray();
```

Comment:
Please add a space around `=`.

Reply:
Also, do we really need to convert it to array? Just realized that this might be the same as before.

Reply:
object is passed by reference, convert it to array and the link will be removed, the we are fine to unset attributes without affect to original object.